app/about_us/aboutCSS/crew.module.css

```diff
@@ -0,0 +1,70 @@
+.crewContainer {
+    display: flex;
+    flex-direction: column;
+  }
+  
+  .sectionIntro {
+    font-size: 1.2rem;
+    text-align: justify;
+    word-spacing: 0.1rem;
+    padding: 3rem 0 2rem 0;
+  }
+  
+  .cardStack {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 3rem;}
```

Comment:
Mind the indentation and code formatting here

Reply:
indentation done